components/project/Zlig.vue

```diff
@@ -10,10 +10,14 @@
 
     <p>
       You can find the currently published version at
-      <a
-        href="https://zlig.net" class="underline transition-colors hover:text-black" target="_blank"
+      <NuxtLink
+        to="https://zlig.net"
+        class="underline transition-colors hover:text-black"
+        target="_blank"
         rel="noopener noreferrer"
-      >https://zlig.net</a>.
+      >
+        https://zlig.net
+      </NuxtLink>.
```

Comment:
Unfortunately, this line breaks give an ugly blank before the dot:

![image](https://github.com/user-attachments/assets/878106d0-f9b9-472e-893e-022a1bd22669)

Do you know how this could be fixed?